packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -381,8 +387,11 @@ export class TestObjectProvider implements ITestObjectProvider {
 		entryPoint: fluidEntryPoint,
 		loaderProps?: Partial<ILoaderProps>,
 		requestHeader?: IRequestHeader,
+		packageEntries: Iterable<[IFluidCodeDetails, fluidEntryPoint]> = [
```

Comment:
That's fine for a hack, but it's not great to have an argument that is not used if further arguments are present.
Not sure how many implementations of this interface are out there, but I'd consider adding new API instead on it

Reply:
This was part of my attempt to do some mock of quorum proposals, feel free to ignore.